src/audio/drc/drc_generic.c

```diff
@@ -526,7 +527,7 @@ static void drc_delay_input_sample_s16(struct drc_state *state,
 	*y = y0;
 }
 
-static void drc_s16_default(const struct comp_dev *dev,
+static void drc_s16_default(struct processing_module *mod,
```

Comment:
Note: `struct drc_comp_data *cd` would have been sufficient but I used mod to avoid circular dependency with drc_func typedef.